examples/wistia/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "wistia",
+  "version": "1.0.0",
+  "description": "Vistia  Widget for contentful",
+  "main": "./src/app.js",
+  "scripts": {
+    "env": "env",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "widget:create": "./node_modules/.bin/better-npm-run widget:create",
+    "widget:update": "./node_modules/.bin/better-npm-run widget:update",
+    "widget:dev": "./node_modules/.bin/better-npm-run widget:dev",
+    "widget:serve": "./node_modules/.bin/better-npm-run widget:serve"
+  },
+  "betterScripts": {
+    "widget:create": {
+      "command": "export CONTENTFUL_MANAGEMENT_ACCESS_TOKEN=$CONTENTFUL_MANAGEMENT_ACCESS_TOKEN && contentful-widget create --space-id $SPACE_ID"
+    },
+    "widget:update": {
+      "command": "export CONTENTFUL_MANAGEMENT_ACCESS_TOKEN=$CONTENTFUL_MANAGEMENT_ACCESS_TOKEN && contentful-widget update --space-id $SPACE_ID --force"
+    },
+    "widget:dev": {
+      "command": "export CONTENTFUL_MANAGEMENT_ACCESS_TOKEN=$CONTENTFUL_MANAGEMENT_ACCESS_TOKEN && contentful-widget update --space-id $SPACE_ID --force --src \"http://localhost:3030/index.html\""
```

Comment:
> > Also use single quotes in the command argument to avoid escaping
> 
> Where exactly?

~~~
 "command": "export ... && contentful-widget update --space-id $SPACE_ID --force --src 'http://localhost:3030/index.html'"
~~~